test/overflow_view_test.dart

```diff
@@ -4,73 +4,6 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:overflow_view/overflow_view.dart';
 
 void main() {
-  test(
```

Comment:
Why removing all these tests?

Reply:
I think it's because `builder` param is now `required`, and can't be null. If we try to pass null as value the compilator will complain about that. We have also removed assertion like `assert(builder != null)` it's already covered by the null-safety feature

Reply:
Oh yeah of course 😅